docs/tutorial/quick-start.md

```diff
@@ -1,22 +1,298 @@
+    ✔ Fixing .gitignore
+
+    We have ATTEMPTED to convert your app to be in a format that electron-forge understands.
+
+    Thanks for using "electron-forge"!!!
+    ```
+
+1. Create a distributable:
+
+    ```sh
+    npm run make
+
+    > my-gsod-electron-app@1.0.0 make /my-electron-app
+    > electron-forge make
+
+    ✔ Checking your system
+    ✔ Resolving Forge Config
+    We need to package your application before we can make it
+    ✔ Preparing to Package Application for arch: x64
+    ✔ Preparing native dependencies
+    ✔ Packaging Application
+    Making for the following targets: zip
+    ✔ Making for target: zip - On platform: darwin - For arch: x64
+    ```
+
+    Electron-forge creates the `out` folder where your package will be located:
+
+    ```plain
+    // Example for MacOS
+    out/
+    ├── out/make/zip/darwin/x64/my-electron-app-darwin-x64-1.0.0.zip
+    ├── ...
+    └── out/my-electron-app-darwin-x64/my-electron-app.app/Contents/MacOS/my-electron-app
+    ```
+
+[node-download]: https://nodejs.org/en/download/
+
+## Learning the basics
+
+This section guides you through the basics of how Electron works under the hood. It aims at strengthening knowledge about Electron and the application created earlier in the Quickstart section.
+
+### Application architecture
+
+Electron consists of three main pillars:
+
+* **Chromium** for displaying web content.
+* **Node.js** for working with the local filesystem and the operating system.
+* **Custom APIs** for working with often-needed OS's native functions.
+
+Developing an application with Electron is like building a Node.js app with a web interface or building web pages with seamless Node.js integration.
+
+#### Main and Renderer Processes
+
+As it was mentioned before, Electron has two types of processes: Main and Renderer.
+
+* The Main process **creates** web pages by creating `BrowserWindow` instances. Each `BrowserWindow` instance runs the web page in its Renderer process. When a `BrowserWindow` instance is destroyed, the corresponding Renderer process gets terminated as well.
+* The Main process **manages** all web pages and their corresponding Renderer processes.
+
+----
+
+* The Renderer process **manages** only the corresponding web page. A crash in one Renderer process does not affect other Renderer processes.
+* The Renderer process **communicates** with the Main process via IPC to perform GUI operations in a web page. Calling native GUI-related APIs from the Renderer process directly is restricted due to security concerns and potential resource leakage.
+
+----
+
+The communication between processes is possible via Inter-Process Communication (IPC) modules: `ipcMain` and `ipcRenderer`.
+
+#### APIs
+
+##### Electron API
+
+Electron APIs are assigned based on the process type. It means that some of the methods can be used from the Main or Renderer processes respectively or both. API documentation indicates the process a particular API method can be used from.
```

Comment:
Process-specific functionality in Electron is typically delineated per module rather than method.

```suggestion
Electron APIs are assigned based on the process type, meaning that some modules can be used from either the Main or Renderer process, and some from both. Electron's API documentation indicates which process each module can be used from.
```